financial-templates-lib/price-feed/CreatePriceFeed.js

```diff
@@ -3,7 +3,7 @@ const { MedianizerPriceFeed } = require("./MedianizerPriceFeed");
 const { CryptoWatchPriceFeed } = require("./CryptoWatchPriceFeed");
 const { UniswapPriceFeed } = require("./UniswapPriceFeed");
 
-const Uniswap = require("../../core/build/contracts/Uniswap.json");
+const Uniswap = require("@umaprotocol/core/build/contracts/Uniswap.json");
```

Comment:
So is this the way to import contract ABI's now?

Reply:
That's the way to get the json from other packages. We _should_ also be able to use  `artifacts.require("@umaprotocol/core/SomeContract")`.